docs/usage/self-hosted-configuration.md

```diff
@@ -684,6 +694,12 @@ Use the `extends` field instead of this if, for example, you need the ability fo
     When Renovate resolves `globalExtends` it does not fully process the configuration.
     This means that Renovate does not have the authentication it needs to fetch private things.
 
+## globalExtendsEager
+
+By default, Renovate resolves and merges `globalExtends` presets _after_ other global config.
+
+But if you set `globalExtendsEager` to `true`, then Renovate will resolve and merge `globalExtends` presets _before_ the global config.
+
```

Comment:
I just realized that we should reverse this default and drop this option in v38.

Maybe better to remove this from the migration?